codegen/can_messages.asciipb

```diff
@@ -0,0 +1,747 @@
+# Message Definitions in plaintext are on the wiki at:
+# https://uwmidsun.atlassian.net/wiki/spaces/ELEC/pages/50003973/CAN+Message+Definitions
```

Comment:
lol this is way outdated, maybe update this with a new wiki page or otherwise just remove this

Reply:
this file should probably be json lol, but that's another PR